components/inputs/input-textarea.js

```diff
@@ -194,6 +197,20 @@ class InputTextArea extends FocusMixin(LabelledMixin(FormElementMixin(SkeletonMi
 		}
 	}
 
+	disconnectedCallback() {
+		super.disconnectedCallback();
+		this.removeEventListener('mouseover', this._handleMouseEnter);
+		this.removeEventListener('mouseout', this._handleMouseLeave);
+		this.removeEventListener('click', this._handleClick);
+	}
+
+	firstUpdated(changedProperties) {
```

Comment:
I know `d2l-input-text` does it this way, but I think these should be in `connectedCallback`. Otherwise, disconnecting will remove them and re-connecting won't add them again (since `firstUpdated` only runs the first time).